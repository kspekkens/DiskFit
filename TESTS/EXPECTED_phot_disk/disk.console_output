 DiskFit (v1.2.2)  Copyright (C) 2015, 2017, Jerry Sellwood and Kristine Spekkens

 This program comes with ABSOLUTELY NO WARRANTY.  It is free software
 and you are welcome to redistribute it under certain conditions.
 See <http://www.gnu.org/licenses/> for details.

 Enter input parameter file name

 Read in input parameters from: EXAMPLE/PHOT/phot_disk.inp                                                                          

Input disk eps: (1-b/a)  0.40
Input disk PA (N->E; deg) =  35.00
Input x-center (data units) =   130.00
Input y-center (data units) =   130.00
Sky, sky sig, gain (ADU) =   800.00   14.14    4.00

 Fitting for:
 disk PA
 disk ellipticity
 disk x-center, y-center
 --> Other attributes fixed to input values

 Surface brightnesses extracted at radii:
    5.0   10.0   15.0   20.0   25.0   30.0   35.0   40.0   45.0   50.0
   55.0   60.0   65.0   70.0   75.0   80.0   90.0  100.0  110.0

 Read in photometric image from:
 EXAMPLE/PHOT/phot.fits

 Number of pixels in selected part of the image:       65536
 Number of pixels used in fit:        7495
   diskpa:   diskel:     xcen:     ycen:
   35.0000  0.400000  130.0000  130.0000
starting iteration    1 best reduced chisq so far: 5.223E+00
   diskpa:   diskel:     xcen:     ycen:
   35.0000  0.400000  130.0000  130.0000
starting iteration    2 best reduced chisq so far: 4.195E+00
   diskpa:   diskel:     xcen:     ycen:
   30.3458  0.299129  130.0000  130.0000
starting iteration    3 best reduced chisq so far: 4.195E+00
   diskpa:   diskel:     xcen:     ycen:
   30.7228  0.299256  130.0000  130.0000

 Done minimization
 Total number of iterations:            3
 Minimum chi^2 found:    4.19463110    

 Wrote:  EXAMPLE/PHOT/OUT/disk.mod.fits
 Wrote:  EXAMPLE/PHOT/OUT/disk.res.fits
 Wrote out: EXAMPLE/PHOT/OUT/disk.out
