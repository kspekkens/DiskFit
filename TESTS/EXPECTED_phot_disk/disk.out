Minimization output, phot
-------------------------
Input files:
EXAMPLE/PHOT/phot_disk.inp
EXAMPLE/PHOT/phot.fits  pixscale  0.50 arcsec/pix
sky:  800.00  skysig:   14.14    gain:    4.00

FITS region:       x-low:      1   y-low:      1 x-range:    256 y-range:    256
FITS sampling:    regrad:  10.00   regpa:  35.00  regeps:   0.40 istpout:      2

Output model and (data-model) residuals files:
EXAMPLE/PHOT/OUT/disk.mod
EXAMPLE/PHOT/OUT/disk.res

Disk toggles:         PA: T     eps: T  center: T non-axi: F    phib: F
phot toggles:     lbulge: F    lr_e: F  lsersn: F blg eps: F bar eps: F

Input values
------------
disk PA, phi_d^prime (deg):           35.00
disk eps:                              0.40
x,y center (data units):             130.00  130.00









No seeing correction applied
No model component smoothing applied
No uncertainties estimated



Best fitting values
-------------------
disk PA, phi_d^prime (deg):           30.72 +/-  0.00
disk eps:                              0.30 +/-  0.00
disk incl (deg):                      45.51 +/-  0.00
x,y center (data units):             130.00 +/-  0.00,  130.00 +/-  0.00











Minimization Details
---------------------
# points Dn used in fit:               7495
# iterations in minimization:             3
Minimum chi^2 found:               4.194631
Degrees of freedom in fit:             7472

Fitted intensities                                         (radii in pixels, intensities in ADU):
       r        npts       Idisk      eIdisk        Ibar       eIbar     Ibulge      eIbulge
---------------------------------------------------------------------------------------------
      5.00    125.04     1868.72        0.00        0.00        0.00        0.00        0.00
     10.00     99.31      935.45        0.00        0.00        0.00        0.00        0.00
     15.00     82.23      894.20        0.00        0.00        0.00        0.00        0.00
     20.00    110.07      650.75        0.00        0.00        0.00        0.00        0.00
     25.00    137.83      475.98        0.00        0.00        0.00        0.00        0.00
     30.00    165.10      372.60        0.00        0.00        0.00        0.00        0.00
     35.00    192.24      280.61        0.00        0.00        0.00        0.00        0.00
     40.00    220.52      207.08        0.00        0.00        0.00        0.00        0.00
     45.00    247.79      171.09        0.00        0.00        0.00        0.00        0.00
     50.00    274.33      137.52        0.00        0.00        0.00        0.00        0.00
     55.00    303.92      112.57        0.00        0.00        0.00        0.00        0.00
     60.00    329.43       90.93        0.00        0.00        0.00        0.00        0.00
     65.00    357.72       76.10        0.00        0.00        0.00        0.00        0.00
     70.00    385.48       61.04        0.00        0.00        0.00        0.00        0.00
     75.00    413.04       49.97        0.00        0.00        0.00        0.00        0.00
     80.00    673.74       39.66        0.00        0.00        0.00        0.00        0.00
     90.00    990.66       28.83        0.00        0.00        0.00        0.00        0.00
    100.00   1101.04       10.01        0.00        0.00        0.00        0.00        0.00
    110.00   1285.52        1.97        0.00        0.00        0.00        0.00        0.00
